ecommerce/products/templates/product_by_type.html

```diff
@@ -0,0 +1,15 @@
+<h1>Products by Type</h1>
+
+<h2>All Products of This Type</h2>
+
+{% for product in product_type %}
+
+<ul>
+    <li>
+        <div>
+            <div><strong>Name: </strong>{{ product.name }}</div>
+            <div><strong>Type: </strong>{{ product.product_type }}</div>
+        </div>
+    </li>
+</ul>
+{% endfor %}
```

Comment:
There's a lot of stuff from your old PR appearing in this pr. To fix this, do `git pull origin develop` and then `git push`. This happens a lot, it just means your local version of `develop` is out of date with your remote (github) version of `develop`

Reply:
Ok cool. I did that and it said Everything up-to-date.